src/bundles/files.js

```diff
@@ -195,12 +193,15 @@ export default {
       const updateProgress = (bytesLoaded) => {
         const progress = Math.round((bytesLoaded / fileSize) * 100)
 
-        dispatch({ type: 'FILES_ADD_PROGRESS', payload: { id: fileId, progress: progress } })
+        dispatch({ type: 'FILES_ADD_PROGRESS', payload: { id: fileName, progress: progress } })
       }
 
-      ipfs.add(stream, { pin: false, progress: updateProgress })
-        .then(addedFile => dispatch({ type: 'FILES_ADD_FINISHED', payload: { id: fileId, hash: addedFile[0].hash } }))
-        .catch(err => dispatch({ type: 'FILES_ADD_FAILED', payload: { id: fileId, error: err.message } }))
+      try {
```

Comment:
I love `async/await` too, but for one liners like this I prefer `then` and `catch` to save keystrokes. It's two lines less and keeps things tidy 😛

Reply:
I just changed it to keep the style streamlined and avoid having multiple kinds of ways of doing the same thing.

Reply:
Alright, good thinking 👍